index.html

```diff
@@ -342,17 +342,17 @@
             </li>
             <li>
               If the algorithm isn't <a>allowed to show a popup</a>, return a
-              <a>Promise</a> rejected with an <a>InvalidAccessError</a> exception
+              promise rejected with an <a>InvalidAccessError</a> exception
               and abort these steps.
             </li>
             <li>
-              If there is already an unsettled <a>Promise</a> from a previous
+              If there is already an unsettled promise from a previous
               call to <code>connect</code> for the same browsing context, return
-              a <a>Promise</a> rejected with an <a>OperationError</a> exception and
+              a promise rejected with an <a>OperationError</a> exception and
               abort all remaining steps.
             </li>
             <li>
-              Let <var>promise</var> be a new <a>Promise</a>.
+              Let <var>promise</var> be a new promise.
```

Comment:
Guess that's when using capital letters for variable names could be beneficial? I bet distinguishing <var>promise</var> and promise is hard when viewing the spec in the browser.